src/azure-cli/HISTORY.rst

```diff
@@ -28,6 +28,10 @@ Release History
 
 * `az storage blob list`: Support OR properties with new api version (#14832)
  
+**SQL**
+
+* Add backup storage redundancy optional parameter to MI create cmdlet (#15071)
+
```

Comment:
We no longer need to write the history manually, it will be generated automatically from the PR title.

Reply:
ok, will remove